lib/matplotlib/axes/_axes.py

```diff
@@ -384,6 +404,207 @@ def legend(self, *args, **kwargs):
     def _remove_legend(self, legend):
         self.legend_ = None
 
+    def inset_axes(self, rect, transform=None, **kwargs):
+        """
+        Add a child inset axes to this existing axes.
+
+        Parameters
+        ----------
+
+        rect : [x0, y0, width, height]
+            Lower-left corner of inset axes, and its width and height.
+
+        transform : `.Transform`
+            Defaults to `ax.transAxes`, i.e. the units of *rect* are in
+            axes-relative coordinates.
+
+        zorder : number
+            Defaults to 5 (same as `.Axes.legend`).  Adjust higher or lower
+            to change whether it is above or below data plotted on the axes.
+
+        Other Parameters
+        ----------------
+
+        kwargs are passed on to the `axes.Axes` child axes.
+
+        Returns
+        -------
+
+        New `.axes.Axes` instance.
+
+        Examples
+        --------
+
+        This example makes two inset axes, the first is in axes-relative
+        coordinates, and the second in data-coordinates::
+
+            fig, ax = plt.suplots()
+            ax.plot(range(10))
+            axin1 = ax.inset_axes([0.8, 0.1, 0.15, 0.15])
+            axin2 = ax.inset_axes([5, 7, 2.3, 2.3], transform=ax.transData)
+
+        """
+        if transform is None:
+            transform = self.transAxes
+        label = kwargs.pop('label', 'inset_axes')
+
+        # This puts the rectangle into figure-relative coordinates.
+        bb = _inset_locator(rect, transform, self)(None, None)
+        # if not set, set to the same zorder as legend
+        zorder = kwargs.pop('zorder', 5)
```

Comment:
Legend.zorder, perhaps?

Reply:
Well, I don't think I want it *tied* to the legend value.  And legend's value is hard coded i.e. not an rcParam....